google-cloud-clients/google-cloud-core/src/main/java/com/google/cloud/ByteArray.java

```diff
@@ -117,7 +117,7 @@ public final void copyTo(ByteBuffer target) {
    * @throws IndexOutOfBoundsException if the target is not large enough to hold the data
    */
   public final void copyTo(byte[] target) {
-    byteString.copyTo(target, 0, 0, length());
+    byteString.copyTo(target, 0);
```

Comment:
I believe this new method is the only option after deprecated four params copyTo method... Correct?